header-footer-grid/templates/row-wrapper-mobile.php

```diff
@@ -21,15 +21,17 @@
 $inner_classes    = 'header-menu-sidebar-inner ' . ( $is_contained ? ' container' : '' );
 $item_attributes  = apply_filters( 'neve_nav_toggle_data_attrs', '' );
 $close_classes    = 'close-sidebar-panel navbar-toggle-wrapper' . ( $close_contained ? ' container' : '' );
-
-$menu_icon_class = apply_filters( 'neve_menu_icon_classes', 'hamburger is-active ' );
+$submenu_style    = row_setting( 'layout', 'slide_left' );
+$button_style     = $submenu_style === 'dropdown' ? 'display:none' : '';
+$focus_class      = $submenu_style === 'dropdown' ? '' : 'ffocus';
+$menu_icon_class  = apply_filters( 'neve_menu_icon_classes', 'hamburger is-active ' );
```

Comment:
`$focus_class` can be appended to the string that is passed to `neve_menu_icon_classes` this way there is no need to `echo` the attribute on L34.